blog/2024-11-26-starting-your-webassembly-developer-loop-with-wash-dev/index.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: "Starting your WebAssembly developer loop with wash dev"
+authors: [ericgregory]
+slug: 2024-11-26-starting-your-webassembly-developer-loop-with-wash-dev
+image: './images/wash-dev.webp'
+date: "2024-11-26"
+description: "In this blog, we'll explain how you can use `wash dev` to start developing WebAssembly components in minutes."
+---
+
+![Starting your WebAssembly developer loop with wash dev](./images/wash-dev.webp)
+
+In [wasmCloud 1.3](/blog/2024-10-20-wasmcloud-1.3-transforms-developer-experience-webassembly-components), we introduced a major refinement on the `wash dev` subcommand that gives WebAssembly component developers a hot-reloading developer loop across all of our supported languages&mdash;and leverages plug-and-play capabilities to make application development even smoother. 
```

Comment:
```suggestion
Alongside [wasmCloud 1.3](/blog/2024-10-20-wasmcloud-1.3-transforms-developer-experience-webassembly-components), we introduced a major refinement on the `wash dev` subcommand that gives WebAssembly component developers a hot-reloading developer loop across all of our supported languages&mdash;and leverages plug-and-play capabilities to make application development even smoother. 
```

supernit because this mostly came the wash version, but I don't think this is a blocker since it points to that blog